README.md

```diff
@@ -1,2 +1,69 @@
 # jinad
+
 Jina Flow Management API
+
+
+## Set up jinad on AWS
+
+1. Create a new instance on AWS and log into the instance
+
+```bash
+ssh -i your.pem ubuntu@ec2-3-17-167-247.us-east-2.compute.amazonaws.com
+```
+
+2. Install the required packages
+
+```bash
+sudo apt-get update
+sudo apt-get -y install python3.8 python3.8-dev python3.8-distutils python3.8-venv python3-pip
+```
```

Comment:
can we dockerize it?

Reply:
Sure. Setting a docker image is the next step.

Reply:
I'm planning to work on this tomorrow with a daemon running, rather than just the server in the background. will update the readme post that